infer/environment.cc

```diff
@@ -1484,6 +1484,10 @@ Environment::processBinding(core::Context ctx, const cfg::CFG &inWhat, cfg::Bind
                 tp.type = core::Types::bottom();
                 tp.origins.emplace_back(ctx.locAt(bind.loc));
             },
+            [&](cfg::VolatileRead &i) {
+                tp.type = core::Types::untypedUntracked();
+                tp.origins.emplace_back(ctx.locAt(bind.loc));
```

Comment:
I'm not sure where this would matter, but should we make `tp` be equal to whatever the type and origins of `i.what` is?